addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -2,7 +2,7 @@
 
 import { _t } from "@web/core/l10n/translation";
 import { getBorderWhite, DEFAULT_BG, getColor, hexToRGBA } from "@web/core/colors/colors";
-import { formatFloat } from "@web/views/fields/formatters";
```

Comment:
When you use the function from numbers.js instead of the one from formatters.js you bypass the `if (value === false) { return ""; }` condition. This makes the graph view crash in Sales (and maybe others, didn't check) because it can send false values to formatFloat

Reply:
https://github.com/odoo/odoo/pull/136113